src/main/java/nextstep/subway/domain/Sections.java

```diff
@@ -25,6 +25,10 @@ public int count() {
         return sections.size();
     }
 
+    public List<Section> getSectionList() {
```

Comment:
이 부분도 마찬가지로 메서드 네이밍에서 특정 자료형을 한정하지 않는 방향으로 고민해봐도 좋을 것 같습니다 :) 
```suggestion
    public List<Section> getSections() {
```